Orm/Xtensive.Orm/Orm/Domain.cs

```diff
@@ -370,7 +370,8 @@ internal async Task<Session> OpenSessionInternalAsync(SessionConfiguration confi
 
     /// <inheritdoc/>
     public IExtensionCollection Extensions { get; private set; }
-    
+    public TaggingBehavior TaggingBehavior { get; internal set; }
```

Comment:
Why? The setting is already in the configuration so use the domain configuration setting it.

Reply:
Simplified it

Reply:
Not so much places use it as I see. Also allowing this property to be changed during Domain instance life is inappropriate.
Do you track it somehow in ST?